README.md

```diff
@@ -24,12 +24,19 @@ run the examples too.
 Building libavif requires [CMake](https://cmake.org/).
 
 No AV1 codecs are enabled by default. Enable them by enabling any of the
-following CMake options:
-
-* `AVIF_CODEC_AOM` - requires CMake, NASM
-* `AVIF_CODEC_DAV1D` - requires Meson, Ninja, NASM
-* `AVIF_CODEC_LIBGAV1` - requires CMake, Ninja
-* `AVIF_CODEC_RAV1E` - requires cargo (Rust), NASM
+following CMake options (e.g. `-DAVIF_CODEC_AOM=ON`):
+
+* `AVIF_CODEC_AOM` for [libaom](https://aomedia.googlesource.com/aom/) (encoder
```

Comment:
Nit: should it be `libaom` or `aom`?

Reply:
The AV1 team at Google refer to this library as "libaom", but I know people outside our team often call it "aom".

If `AVIF_CODEC_AOM` is on, one can further use `AVIF_CODEC_AOM_DECODE` and `AVIF_CODEC_AOM_ENCODE` because libaom can be used as both a decoder and an encoder.